www/apps/book/app/basics/modules-and-services/page.mdx

```diff
@@ -74,7 +74,9 @@ In `medusa-config.js`, add a `modules` property and pass to it your custom modul
 
 ```js title="medusa-config.js" highlights={[["4", "helloModuleService", "The key of the main service to be registered in the Medusa container."]]}
```

Comment:
```suggestion
```js title="medusa-config.js" highlights={[["7", "helloModuleService", "The key of the main service to be registered in the Medusa container."]]}
```